packages/instrumentor/plugins/functionHooks.test.ts

```diff
@@ -20,17 +20,69 @@
 import { instrumentAndEvalWith } from "./testhelpers";
 import { functionHooks } from "./functionHooks";
 import * as hooking from "@jazzer.js/hooking";
+import { Hook, trackedHooks } from "@jazzer.js/hooking";
 
 const expectInstrumentationEval = instrumentAndEvalWith(
 	functionHooks("pkg/lib/a")
 );
 
 registerHookManagerGlobally();
 
+//eslint-disable-next-line @typescript-eslint/no-explicit-any
+function withDebug(fn: () => void): jest.Mock<any, any> {
+	const log = console.log;
+	const mock = jest.fn();
+	console.log = mock;
+	process.env["JAZZER_DEBUG"] = "1";
+	try {
+		fn();
+	} finally {
+		console.log = log;
+		delete process.env["JAZZER_DEBUG"];
+	}
+	return mock;
+}
+
+function expectLogHooks(
+	//eslint-disable-next-line @typescript-eslint/no-explicit-any
+	mock: jest.Mock<any, any>,
+	callsites: number,
+	hookType: string,
+	hookName: string
+) {
+	expect(mock).toBeCalledTimes(callsites);
+	if (callsites > 0) {
+		const hookTp: string = mock.mock.calls[0][1];
+		expect(hookTp).toEqual(hookType);
+		const hookNm: string = mock.mock.calls[0][3];
+		expect(hookNm).toEqual(hookName);
+	}
+}
+
+function expectTrackedHooks(tracker: Set<string>, entries: string[]) {
+	expect(entries.every((e) => [...tracker].includes(e))).toBeTruthy();
```

Comment:
What do you think of this:

```suggestion
	expect(entries.every((e) => tracker.has(e))).toBeTruthy();
```